### PR TITLE
[BACKLOG-12885]-Create and dev validate EMR 5.2 shim against Pentaho 7.0

### PR DESCRIPTION
**Pull request by VasilinaTerehova**

### CODE REVIEW

**Comment by VasilinaTerehova** [comment]
@dkincade @hudak could you please take a view?

**Comment by dkincade** [comment]
Looks good to me ... @buildguy @lgrill-pentaho @smaring please review the build / properties aspect of this PR against the 7.0 branch ...

File: emr52/build.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--===========================================================================
+  This is the build file for the Hadoop EMR 4.$ Shim modules.
+
+  See ../build-res/subfloor.xml for more details
+============================================================================-->
+<project name="pentaho-hadoop-shims-emr52" basedir="." default="dist"
```
Comment (lgrill-pentaho) [inline emr52/build.xml]: If this was <project name'${ivy.artifact.id}" ... You wouldn't have to change it every time.

File: emr52/build.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  xmlns:ivy="antlib:org.apache.ivy.ant" >
+  <description>
+    This build file is used to create the Hadoop EMR 4.1 Shim module for the Big Data plugin
```
Comment (lgrill-pentaho) [inline emr52/build.xml]: It's not EMR 41 right?

Comment (VasilinaTerehova) [inline emr52/build.xml]: yes, will fix it now

Comment (VasilinaTerehova) [inline emr52/build.xml]: fixed, thank you for the note

File: emr52/build.xml (hunk 1 above)
Comment (VasilinaTerehova) [inline emr52/build.xml]: fixed

**Comment by VasilinaTerehova** [comment]
@dkincade all the notes have been fixed

**Pull request merged**